test.rb

```diff
@@ -0,0 +1 @@
+p 'hello world!'
```

Comment:
I’m GitHub I’m fast safe and secure on internet connection and also an icon for browser